locales/de/subtitles/giacomina-castagnetti/giacomina-castagnetti-01.yml

```diff
@@ -3,40 +3,41 @@ de:
   giacomina-castagnetti:
     chapter-1:
       c: 1
-      title: Family, social and political context
+      title: Familie, sozialer und politischer Kontext
       subtitles:
         sequence-1:
           s: 1
           from: 00:00:00,18
           to: 00:00:03,11
-          text: My name is Giacomina Castagnetti.
+          text: Mein Name ist Giacomina Castagnetti.
         sequence-2:
           s: 2
           from: 00:00:03,11
           to: 00:00:08,13
-          text: I always accept when I’m asked to talk about my life.
+          text: Ich akzeptiere es immer, wenn mich jemand nach meiner
```

Comment:
With original transcript at hand

> Accetto sempre volentieri quando mi chiedono di raccontare la mia storia

I would say a close translations could be 

> Ich nehme es immer gerne an, wenn ich gefragt werde, meine (Lebens)Geschichte zu erzählen.

But some Italian native speakers would be helpful :)